content/java/concepts/calendar/terms/add/add.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: '.add()'
+Description: 'Adds or subtracts a specified amount of time to/from the given Calendar field.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.add()`** method of the `Calendar` class is used to manipulate and modify a `Calendar` instance by adding or subtracting a specified amount of time to/from a specfic Calendar field.
+
+## Syntax
+
+```pseudo
+myCalendar.add(field, amount)
+```
+
+- `field`: An integer constant representing the field to which the amount of time should be added or subtracted. This can be one of the following constants:
+    - `Calendar.YEAR`: Adds or subtracts the specified number of years.
```

Comment:
I think these fields are pretty self-explanatory. I would suggest only listing a few as examples of what a field value could be, or replace the existing descriptions with more detailed information that may not be as obvious such as the range for the given field.